src/html5/tokenizer.rs

```diff
@@ -50,6 +51,21 @@ pub struct Tokenizer<'stream> {
     pub error_logger: Rc<RefCell<ErrorLogger>>,
 }
 
+/// This struct is a gateway between the parser and the tokenizer. It holds data that can be needed
+/// by the tokenizer in certain cases. See https://github.com/gosub-browser/gosub-engine/issues/230 for
+/// more information and how we should refactor this properly.
+pub struct ParserData {
+    pub adjusted_node_namespace: String,
+}
```

Comment:
I suspect this will evolve into something else at some point.  But seems fine as it is for now.

Reply:
yes.. this is a dirty hack that we have to get rid of.. there is bidirectional communication between the tokenizer and parser which we can avoid when introducing a tree-builder system. Then, both the tokenizer and parser will talk to it, but not to eachother, which makes the dependencies work again.